backend/baseclass.pm

```diff
@@ -265,22 +276,16 @@ sub run_capture_loop ($self, $timeout = undef) {
 # This is not sliding buckets. All the IDs inside the bucket must be over the limit!
 sub check_select_rate ($buckets, $wait_time_limit, $hits_limit, $id) {
     my $time = gettimeofday;
-    my $lower_limit = $buckets->{TIME} //= $time;
-    my $upper_limit = $lower_limit + $wait_time_limit;
-    if ($time > $upper_limit) {
-        $buckets->{TIME} = $time;
-
-        # This is to give the opportunity to recover, if the reboot/restart is slow
-        for (keys %{$buckets->{BUCKET}}) {
-            if ($buckets->{BUCKET}{$_} < $hits_limit) {
-                $buckets->{BUCKET} = {$id => 1};
-                return 0;
-            }
-        }
-
-        return 1;
+    my $bucket = $buckets->{$id} //= {cnt => 0, time => $time};
+
```

Comment:
```suggestion
    my $bucket = $buckets->{$id} //= {cnt => 0, time => $time};
```

Reply:
I can't spot the difference. What did you suggest to change? :D

Reply:
:question:

Reply:
Remove the empty line?

Reply:
Yes, I wanted to suggest removing the empty line.